proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java

```diff
@@ -682,9 +683,13 @@ public void handle(Commands commands) throws Exception
     @Override
     public void handle(ServerData serverData) throws Exception
     {
-        serverData.setMotd( null );
-        serverData.setIcon( null );
-        con.unsafe().sendPacket( serverData );
+        if ( enforceSecure ^ ( enforceSecure = serverData.isEnforceSecure() ) )
```

Comment:
Uh downstream bridge changes each server, so this should be in the UserConnection class. Also ^= is a thing isnt it?

Reply:
`^=` is not equivalent to the code above.
it will give you the resulting `enforceSecure` boolean value and not if there was a difference.
additionally enforceSecure value will be wrong.

Reply:
Thanks